packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md

```diff
@@ -81,13 +80,23 @@ Limits are applied on a per-recipient basis and vary by email type. The limit fo
 
 ### Per-user limits
 
-API endpoints that take username and password credentials, including the [Authentication API](/docs/reference/api/authn/) and the [OAuth 2.0 resource owner password flow](/docs/guides/implement-password/), have a per-username rate limit to prevent brute force attacks with the user's password:
+API endpoints that take username and password credentials, including the [Authentication API](/docs/reference/api/authn/), the [OAuth 2.0 resource owner password flow](/docs/guides/implement-password/), and SMS and Call factor endpoints, have a per-username rate limit to prevent brute force attacks with the user's password:
```

Comment:
Same as above - this is a per-user limit, but it isn't a username/password operation.